front-page.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/**
+ * Display front-page
+ *
+ * @package Assignment
+ */
+
+use AssignmentVendor\EightshiftLibs\Helpers\Components;
+
+get_header();
+
+if (have_posts()) {
+	while (have_posts()) {
+		the_post();
+		echo Components::render( // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+			'heading',
+			[
+				'headingContent' => the_title('', '', false),
```

Comment:
A bit simpler version ;)

```suggestion
				'headingContent' => get_the_title(),
```